Changelog.md

```diff
@@ -9,6 +9,8 @@
 
 -   Chart frame trotting for slower machines.
 -   Clearing session files on close in some special cases.
+-   When recording indefinitely **Disk full trigger** stop recording when disk
+    is full
```

Comment:
```suggestion
-   Behavior of **Disk full trigger** when recording for an indefinite period of time. It now correctly stops recording when the disk is full.
```

Reply:
Maybe use "sampling" instead of "recording"?